distribution/std-lib/Standard/src/Base/Data/Any/Extensions.enso

```diff
@@ -121,28 +201,34 @@ Any.<| ~argument = this argument
    Arguments
    - function: The function to apply to `this`.
 
+   ? `|>` or `.`?
+     The eagle-eyed reader will notice that the operator dot (`.`) is very
+     similar to the operator `|>`. In Enso, with the variable precedence of
+     operators, this makes perfect sense. In general, we recommend using `.`.
+     However, there are some contexts where variable precedence might be unclear
+     or confusing, and for those times we recommend using `|>`.
+
    > Example
-     Applying a function in a pipeline.
-         1 |> (* 2)
+     Applying multiple functions in a pipeline to compute a number and transform
+     it to text.
+         1 |> (* 2) |> (/ 100) |> .to_text
 Any.|> : (Any -> Any) -> Any
-Any.|> function = function this
-
-## Composes two functions together.
+Any.|> ~function = function this
```

Comment:
Why is the `function` value suspended here? We immediately apply it to `this` so I see no situation where this could help.

Or is this for the case where `this` is a dataflow error? (I'm not sure what are the semantics in that case)